EasyThemes.php

```diff
@@ -221,16 +216,12 @@ protected function prepareBackendNavigationArray($blnForceToShowLabel = false)
             if (isset($GLOBALS['TL_EASY_THEMES_MODULES'][$strModule]['href'])) {
                 $href = sprintf($GLOBALS['TL_EASY_THEMES_MODULES'][$strModule]['href'], $intThemeId);
             } else if (isset($GLOBALS['TL_EASY_THEMES_MODULES'][$strModule]['href_fragment'])) {
-                $href = $url . '?do=themes&amp;' . $GLOBALS['TL_EASY_THEMES_MODULES'][$strModule]['href_fragment'] . '&amp;id=' . $intThemeId;
+                $arrHrefFragment = explode('=', $GLOBALS['TL_EASY_THEMES_MODULES'][$strModule]['href_fragment']);
+                $href = $router->generate('contao_backend', ['do' => 'themes', $arrHrefFragment[0] => $arrHrefFragment[1], 'id' =>  $intThemeId]);

```

Comment:
@qzminski I'm pretty sure and i tested it ,we don't need it anymore. 
I have removed the part where &amp; is used in the code.
What makes you thinking we still need it?

Reply:
I am not sure if Contao ampersands URLs automatically when adding them to the backend menu. Since `&amp;` was used before, I think we should still ampersand them, as router generates decoded URLs as far as I am concerned. And for the `href` attribute (or any other) we should always use `&amp;` instead of just `&`.